app/src/main/res/raw/japan.json

```diff
@@ -0,0 +1 @@
 1, 2, 11]}, {"name": "red_king_crab", "months": [0, 1, 10, 11]}, {"name": "rice", "months": [8, 9]}, {"name": "righteye_flounder", "months": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11]}, {"name": "rocket", "months": [3, 4, 5, 9, 10, 11]}, {"name": "royal_fern", "months": [3, 4, 5]}, {"name": "sailfin_sandfish", "months": [9, 10, 11]}, {"name": "saled_salmon_roe", "months": [8, 9, 10]}, {"name": "salmon", "months": [8, 9, 10]}, {"name": "sea_bass", "months": [5, 6, 7]}, {"name": "sea_bream", "months": [1, 2, 3]}, {"name": "sea_squirt", "months": [5, 6]}, {"name": "sea_urchin", "months": [4, 5, 6, 7]}, {"name": "sesame", "months": [8, 9]}, {"name": "shimeji", "months": [8, 9, 10]}, {"name": "shishamo_smelt", "months": [10, 11]}, {"name": "shishito_pepper", "months": [5, 6, 7]}, {"name": "shitake", "months": [2, 3, 4, 8, 9, 10]}, {"name": "sichuan_pepper", "months": [3, 4, 5, 9, 10]}, {"name": "sillago", "months": [5, 6, 7]}, {"name": "silver-stripe_round_herring", "months": [4, 5]}, {"name": "skipjack_tuna", "months": [4, 5, 8, 9]}, {"name": "small_red_bean", "months": [0, 1, 9, 10, 11]}, {"name": "snow_crab", "months": [0, 1, 2]}, {"name": "snow_pea", "months": [3, 4, 5]}, {"name": "soy_pulp", "months": [0, 1]}, {"name": "spear_squid", "months": [0, 1, 2, 11]}, {"name": "spinach", "months": [0, 1, 10, 11]}, {"name": "spiny_lobster", "months": [0, 9, 10, 11]}, {"name": "splendid_alfonsino", "months": [0, 1, 11]}, {"name": "strawberry", "months": [1, 2, 3, 4]}, {"name": "striped_mullet", "months": [0, 9, 10, 11]}, {"name": "sudachi", "months": [7, 8, 9]}, {"name": "summer_orange", "months": [2, 3, 4]}, {"name": "sumo_mandarin", "months": [1, 2, 3]}, {"name": "sweet_potato", "months": [8, 9, 10, 11]}, {"name": "sweetfish", "months": [5, 6, 7]}, {"name": "thread-sail_filefish", "months": [5, 6, 7]}, {"name": "threeleaf_arrowhead", "months": [0, 1, 10, 11]}, {"name": "tiger_prawn", "months": [5, 6, 7, 8]}, {"name": "tomato", "months": [5, 6, 7]}, {"name": "tree_onion", "months": [2, 3]}, {"name": "trough_shell", "months": [1, 2, 3]}, {"name": "tuna", "months": [0]}, {"name": "turban_shell", "months": [4, 5, 6, 7]}, {"name": "turnip", "months": [2, 3, 4, 9, 10]}, {"name": "udo", "months": [2, 3, 4]}, {"name": "uncured_onion", "months": [2, 3, 4]}, {"name": "wakame", "months": [2, 3, 4]}, {"name": "water_cress", "months": [2, 3, 4, 5]}, {"name": "water_dropwort", "months": [0, 1, 2, 3, 11]}, {"name": "watermelon", "months": [6]}, {"name": "wax_gourd", "months": [6, 7, 8, 9]}, {"name": "whelsh_onion", "months": [0, 1, 2, 10, 11]}, {"name": "whitebait", "months": [2, 3, 4, 8, 9]}, {"name": "wild_rocambole", "months": [3, 4, 5]}, {"name": "winter_mushroom", "months": [9, 10]}, {"name": "yam", "months": [0, 3, 4, 10, 11]}, {"name": "yesso_scallop", "months": [0]}, {"name": "young_ginger", "months": [5, 6, 7]}, {"name": "young_japanese_amberjack", "months": [0, 10, 11]}, {"name": "yuzu", "months": [6, 7, 9, 10, 11]}, {"name": "zucchini", "months": [5, 6, 7, 8]}]}
```

Comment:
I'm still wondering about some of the very specific ingredients. Did you verify that these are terms used in actual recipes (online or in books)?

For example we have multiple types of ginger, onions, amberjack etc. So do people actually use these specific ingredients when writing recipes, like for example "3 uncured onions" or "1 young Japanese amberjack"? This would be important to know so that seasonal recipes can be identified correctly in the app.